terraform-plans/configs/openstack-exporter-operator_main.tfvars

```diff
@@ -53,4 +53,11 @@ templates = {
       repository = "openstack-exporter-operator"
     }
   }
+  tox = {
```

Comment:
can you apply this template (and also pyproject.toml) to the project first, then port back here

Reply:
My plan was to merge this once approved, and then resolve any issues in the target repos on the automated PRs (similar to what we did with pyproject.toml). This won't work out of the box for some projects, but we can work on any refactoring required on the automated PRs.

Reply:
Got it

Reply:
(I'm not sure if this is the best way?) Either way, it would be good to agree on a template first, and then we have a standard base to apply to all the repositories.